Home Page Content

```diff
@@ -0,0 +1,64 @@
+
+
+For Property Managers, By Property Managers …
+Welcome To The World Of Loca
+
+
+
+
+
+An Advanced Web-based, Open Source Management Application Created Specifically for Rental Property Owners, Landlords and Managers
+  
+Available for Windows, Mac and Linux Users.
+
+Try out Our Demo Here
+
+Loca is a simple but professional, user-friendly and eye-catching Rental Property Management Application.
+
+Professional Property Management, The Stress-Free Way
+It is designed to give you the crucial advantage, customizable solutions and necessary accounting intelligence to help you make sound rental property management decisions and control your rental property expenses effectively.
+
+Best Of All, It’s Totally Free To Use… No Hidden Charges, No cutthroat upfront fees, Nothing!
```

Comment:
So far, I haven't deployed Loca for business purposes. I don't commercialize this app.
What I understand of your lines above, is that I offer a ready to use solution for professional property managers.
Unfortunately, it is not the case so far. You have to clone my code, do some customizations here and there, and run it on a server.